runtime/inspector/mod.rs

```diff
@@ -405,26 +55,12 @@ enum PollState {
 
 pub struct DenoInspector {
   v8_inspector_client: v8::inspector::V8InspectorClientBase,
-  v8_inspector: v8::UniquePtr<v8::inspector::V8Inspector>,
-  sessions: RefCell<InspectorSessions>,
+  // TODO(bartlomieju): make this field private
+  // TODO(bartlomieju): could probably be v8::UniqueRef instead of UniquePtr
+  pub v8_inspector: Rc<RefCell<v8::UniquePtr<v8::inspector::V8Inspector>>>,
+  sessions: RefCell<SessionContainer>,
   flags: RefCell<InspectorFlags>,
   waker: Arc<InspectorWaker>,
-  _canary_tx: oneshot::Sender<Never>,
-  pub server: Option<Arc<InspectorServer>>,
```

Comment:
Removed reference to `server` completely - I tried `poll_sessions()` without short-circuit if there's no server and it still works. This will help decouple inspector instance from the WS server which will only be available in `runtime` crate.

Reply:
Seems good to me!